src/Execution/Executor.php

```diff
@@ -114,8 +114,8 @@ public function execute(): ?array
             $result = $operation->getOperation() === 'mutation'
                 ? $this->executeFieldsSerially($objectType, $rootValue, $path, $fields)
                 : $this->executeFields($objectType, $rootValue, $path, $fields);
-        } catch (\Throwable $ex) {
-            $this->handleError(new ExecutionException($ex->getMessage(), $fields, null, null, null, null, $ex));
+        } catch (ExecutionException $ex) {
```

Comment:
Now \Throwable and other exceptions won't ever be caught?

Reply:
This should be correct, `\Throwable`s are already elsewhere wrapped after `ExecutionException`s.

Reply:
What we can do is add another `catch` clause, just to be sure though. Good point.